func/extractLinks.ts

```diff
@@ -0,0 +1,20 @@
+import cheerio from "cheerio"
+import { writeLog } from "../scraper"
+/**
+ * Extract Links:
+ * Limit the number of extracted links per page to a configurable number (e.g., 10).
+ * @param url - string of url
+ */
+export const extractLinks = async (html: string, breadth: number) => {
```

Comment:
"breadth" makes sense in the context of the parent/grandparent function that calls this

Here it should probably be called something like maxLinks.